src/go/cmd/vm.go

```diff
@@ -123,6 +123,68 @@ func newVMResumeCmd() *cobra.Command {
 	return cmd
 }
 
+func newVMRestartCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "restart <experiment name> <vm name>",
+		Short: "Restart a running, paused, or powered off VM",		
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) != 2 {
+				return fmt.Errorf("Must provide an experiment and VM name")
+			}
+
+			var (
+				expName = args[0]
+				vmName  = args[1]
+			)
+
+			if err := vm.Restart(expName, vmName); err != nil {
+				err := util.HumanizeError(err, "Unable to restart the "+vmName+" VM")
+				return err.Humanized()
+			}
+
+			fmt.Printf("The %s VM in the %s experiment was restarted\n", vmName, expName)
+
+			return nil
+		},
+	}
+
+	return cmd
+}
+
+func newVMResetDiskCmd() *cobra.Command {
+	desc := `Resets the disk state to the initial pre-boot disk state for a running or powered off VM
+	
+  Used to reset the disk state for the first disk for a running or powered off virtual machine for a specific 
+  experiment.  The VM's snapshot flag must be set to true in order to use this command.`
+
+	cmd := &cobra.Command{
+		Use:   "resetdisk <experiment name> <vm name>",
```

Comment:
@eric-c-wood can we hyphenate this subcommand name to keep things consistent?

Reply:
@activeshadow hyphen has been added.